packages/node-modules-inspector/src/app/components/display/VersionWithUpdates.vue

```diff
@@ -24,7 +30,7 @@ const versionDiff = computed(() => {
         <div v-if="latest.version === version" text-sm op50>
```

Comment:
Maybe here instead? If it's newer, we still shows it's up to date

Reply:
> Maybe here instead? If it's newer, we still shows it's up to date

Got you, resolved.